site/en/tutorials/python_quickstart.ipynb

```diff
@@ -141,7 +141,7 @@
     },
```

Comment:
<p>"which is often mentioned in terms of" -&gt; "which can be described in terms of"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
> @8bitmp3 makes a good suggestion. Feel free to make this change and ignore mine below.

Ignoring this change . Thank you.